packages/contracts-ops/contracts/test/RebootHome.t.sol

```diff
@@ -28,11 +30,16 @@ contract HomeRebootTest is RebootTest, HomeTest {
             coreAttributePath(localDomainName, "home.implementation")
         );
         reloadConfig();
-        pushSingleUpgrade(homeUpgrade(localDomainName));
+        pushSingleUpgrade(homeUpgrade(localDomainName), localDomainName);
         prankExecuteRecoveryManager(
             address(getGovernanceRouter(localDomainName)),
             getDomainNumber(localDomainName)
         );
+        // assert beacon has been upgraded to harness
+        (, bytes memory result) = address(homeUpgrade(localDomainName).beacon)
+            .staticcall("");
+        address _current = abi.decode(result, (address));
+        assertEq(_current, address(homeHarnessImpl));
```

Comment:
no asserts in methods called in setup. consider naming methods called in setup `setUp_doThing` so that we know which they are

Reply:
https://github.com/nomad-xyz/monorepo/pull/545/commits/a50e36eac21055df15997d24735ae1ce33d63b64